src/lib/stores/labs-mode.ts

```diff
@@ -0,0 +1,3 @@
+import { persistStore } from './persist-store';
+
+export const labsMode = persistStore('labsMode', false, true);
```

Comment:
Might be nice if we could do something like this 

```js
const labsParam = derived([page], ([$page]) =>
  $page.url.searchParams.get('labs'),
);

const updateLabsMode: StartStopNotifier<boolean> = (set) => {
  return labsParam.subscribe((labs) => {
    if (labs === 'true') {
      set(true);
    } else if (labs === 'false') {
      set(false);
    }
  });
};
```
with the `persistStore` so we don't have to be on the same page where the `LabsModeGuard` is to update the setting.